Removed Preview-oriented language in Time Series Insights Overview

kfile opened the pull request:
Removed "In preview" and other references to preview since this is going to GA. Also fixed a grammar problem for line 40.

azurecla commented:
Hi __@MisinformedDNA__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

rmcmurray commented:
I agree, it's probably best to reword this as "The commands listed above will perform the following actions"

deneha commented:
@MisinformedDNA Thank you for your contribution.  We've merged the new content.

kfile committed:
Removed Preview-oriented language

Removed "In preview" and other references to preview since this is going to GA. Also fixed a grammar problem for line 40.
